openwisp_monitoring/device/models.py

```diff
@@ -1,12 +1,16 @@
 from django.contrib.contenttypes.fields import GenericRelation
 from swapper import get_model_name, swappable_setting
 
-from openwisp_controller.config.models import Device
+# TODO:N Fix me
+from openwisp_controller.config.models import Device as BaseDevice
```

Comment:
For some reason, this works but swappper.load_model is not working because "Model's are not loaded". I am looking into it if it's possible.